client/src/App.tsx

```diff
@@ -25,6 +26,7 @@ function App(): JSX.Element {
                 <Route exact path="/dashboard">
                   <Dashboard />
                 </Route>
+                <Route exact path="/user/:userId" component={Profile} />
```

Comment:
A logged in user will only have one profile, what is the reason we have route parameters here?